dist/templates/widget-cookie-declaration/consent-cookie-declaration.html

```diff
@@ -1,50 +1,48 @@
 <link rel="stylesheet" href="../templates/widget-cookie-declaration/consent-cookie-declaration.css">
-<div class="cooi-declaration-wrapper">
+<div class="coi-cookie-declaration">
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
-  <span class="cooi-page-title">{{{translations.cookies_overview}}}</span>
-  <div class="cooi-cookies-info">
-    <div class="cooi-about-owner">
-      <div class="owner-info">
-      </div>
-      <div class="cooi-website-domains">{{website_domains}}</div>
-      <div class="cooi-last-scan">
-        <span class="cooi-last-scan-text">{{last_updated}}</span>
-        <span class="cooi-last-scan-date">{{last_scan_date}}</span>
+  <span class="coi-cookie-declaration__page-title">{{{translations.cookies_overview}}}</span>
+  <div class="coi-cookie-declaration__cookies-info">
+    <div class="cookies-info__about-owner">
+      <div class="cookies-info__website-domains">{{website_domains}}</div>
```

Comment:
Use 3 brackets when displaying something, you will not escape the values then. You should use two when using the value as a HTML property.